test/lit/i31-new.wast

```diff
@@ -1,6 +1,6 @@
 ;; NOTE: Assertions have been generated by update_lit_checks.py and should not be edited.
 
-;; Check that we still parse the legacy i31.new instruction.
+;; Check that we still parse the legacy ref.i31 instruction.
```

Comment:
Is this test still needed? `ref.i31` is not legacy.

Reply:
Oh, oops. Will delete.